applications/xquare-user-application/src/pages/addonpage.tsx

```
@@ -0,0 +1,106 @@
+          <AddonItem
+            title={item.title}
+            domain={item.domain}
+            type={item.type}
+            description={item.description}
+            traefix={Number(item.traefix)}
+            health={Number(item.health)}
+            lastdeploy={item.lastdeploy}
+            lastbuild={item.lastbuild}
+            charge={item.charge}
+          />
+        ))}
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)